src/Microsoft.DotNet.Build.Tasks/PackageFiles/VersionTools.targets

```diff
@@ -13,6 +13,10 @@
     <BuildInfoCacheDir>$(ToolsDir)BuildInfoCache/</BuildInfoCacheDir>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
Why the difference from #1686 here?  The reasoning for putting the property inside the target is that it gives everyone a chance to set the property before the target runs.  If you set the property in a property group, someone that is imported after this targets file would have to overwrite the default and couldn't do the `== ''` check.

Reply:
Ah, good catch, this was pretty messy in the conflict resolution and I somehow didn't think to check the original state when deciding where to put it. No reason, putting it back.